<ui version="4.0" >
 <class>plotWindow</class>
 <widget class="QWidget" name="plotWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Plot Window</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Plot</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout" >
       <item row="0" column="0" >
        <widget class="MPL_Widget" native="1" name="plotWidget" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <layout class="QGridLayout" name="_2" >
         <item row="0" column="1" >
          <widget class="QLabel" name="label_3" >
           <property name="font" >
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text" >
            <string>X Axis</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2" >
          <widget class="QLabel" name="label_4" >
           <property name="font" >
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text" >
            <string>Y Axis</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="min_label" >
           <property name="text" >
            <string>Minimum:</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QLineEdit" name="xmin_lineEdit" >
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="2" >
          <widget class="QLineEdit" name="ymin_lineEdit" />
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="max_label" >
           <property name="text" >
            <string>Maximum:</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QLineEdit" name="xmax_lineEdit" />
         </item>
         <item row="2" column="2" >
          <widget class="QLineEdit" name="ymax_lineEdit" />
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <attribute name="title" >
       <string>Data Table</string>
      </attribute>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="CustomTable" name="peakTable" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Options</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MPL_Widget</class>
   <extends>QWidget</extends>
   <header>mpl_pyqt4_widget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CustomTable</class>
   <extends>QTableWidget</extends>
   <header>customTable</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
